aztk/upload_node_scripts.py

```diff
@@ -24,8 +24,11 @@ def zipdir(path, ziph):
         relative_folder = os.path.relpath(base, path)
         for file in files:
             if __includeFile(file):
-                with io.open(os.path.join(base, file), 'r') as f:
-                    ziph.writestr(os.path.join(relative_folder, file), f.read().replace('\r\n', '\n'))
+                if (file.endswith(".jar")):
```

Comment:
we should watch for that one. probably have something that check if the file is text vs binary and replace new lines only then

Reply:
This is going to change once the SDK is merged anyways. So if it works now it should be fine.